packages/trader/src/App/Containers/SettingsModal/settings-chart.jsx

```diff
@@ -1,4 +1,4 @@
-import { Checkbox, RadioGroup } from '@deriv/components';
```

Comment:
Sorry for too much cleaning, as my laptop is too slow with few resources, `npm run test` throw an error of timeout, and I cannot check these issue with `npm run test`